### PR TITLE
Sql handling

### PR DESCRIPTION
**Pull request by ZhangMan1126**
update interpolateSqlString with JSqlParser
remove comments
as investigated, no need to handle prepared statements of zebra

### CODE REVIEW

File: client-java/instrumentation/pom.xml [hunk 1]
```diff
@@ -34,6 +34,11 @@
             <artifactId>asm-commons</artifactId>
         </dependency>
 
+        <!--  handle sql command      -->
+        <dependency>
+            <groupId>com.github.jsqlparser</groupId>
+            <artifactId>jsqlparser</artifactId>
+        </dependency>
```
Comment (arcuri82) [inline client-java/instrumentation/pom.xml:41]: if you add dependencies to `instrumentation` those must be shaded. see bottom of pom file for `maven-shade-plugin`

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java [hunk 2]
```diff
@@ -145,11 +190,9 @@ private static String handlePreparedStatement(PreparedStatement stmt) {
...
         }
 
-        /*
-            handle db middleware, ie, zebra
-         */
+        // just check whether there exist any new update
         if (fullClassName.startsWith("com.dianping.zebra")){
-            sql = extractSqlFromZebraPreparedStatement(stmt);
+            throw new IllegalArgumentException("unsupported type for zebra: " + fullClassName);
         }
```
Comment (arcuri82) [inline client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java:196]: why throwing an exception here?  wasn't this checked with `checkZebraPreparedStatementWrapper`?  or should rather be an `assert` here?

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java (hunk 2 above)
Comment (ZhangMan1126) [reply client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java:196]: all zebra prepared statements should be handled before this line (see 155).
if this line is reached, it means that there exist new types of prepared statements, then throw the exception with the info.

File: client-java/instrumentation/pom.xml (hunk 1 above)
Comment (ZhangMan1126) [reply client-java/instrumentation/pom.xml:41]: thanks! updated

File: client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java (hunk 2 above)
Comment (arcuri82) [reply client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java:196]: ok. can u add this clarification as a code comment?

Comment (ZhangMan1126) [reply client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/PreparedStatementClassReplacement.java:201]: the comment is modified with more info now. thanks.

**Pull request merged**